src/fps/proposal/IProposal.sol

```diff
@@ -0,0 +1,62 @@
+pragma solidity 0.8.15;
+
+import {AddressRegistry as Addresses} from "src/fps/AddressRegistry.sol";
+
+abstract contract IProposal {
+    /// @notice proposal name, e.g. "BIP15".
+    /// @dev override this to set the proposal name.
+
+    function name() external view virtual returns (string memory);
+
+    /// @notice proposal description.
+    /// @dev override this to set the proposal description.
```

Comment:
I don't even think we will ever use this, right? Our tasks aren't onchain and the `README.md` files associated with each task should give us all the information we need to understand the context of the given upgrade task.

Reply:
I see this file has been removed, resolving this thread.